internal/notifications/view.go

```diff
@@ -70,26 +70,31 @@ func (n Notifications) String() string {
 	for _, n := range n {
 		out += fmt.Sprintf("%s\n", n)
 	}
+
 	return out
 }
 
 func (n Notifications) Visible() Notifications {
 	visible := Notifications{}
+
 	for _, n := range n {
 		if n.Visible() {
 			visible = append(visible, n)
 		}
 	}
+
 	return visible
 }
```

Comment:
What linter lead to add so many line feed?

I'm curious

Is it a simple gofumpt formating or something else?

Reply:
It could have also been https://github.com/ultraware/whitespace or https://github.com/bombsimon/wsl

Reply:
Thanks